### PR TITLE
Ep4

### PR DESCRIPTION
**Pull request by ryanburgess**
I've started the episode details for episode 4 and I've added Sarah to the list of panelists.

@bittersweetryan do you mind taking a look.

### CODE REVIEW

File: content/episodes.json [hunk 1]
```diff
@@ -1,5 +1,122 @@
...
+      "Jem Young",
+      "Ryan Anklam",
+      "Ryan Burgess",
+      "Brian Holt",
+      "Augustus Yuan",
+      "Sarah Federman"
+    ],
+    "guests": [
+    ],
+    "links": [
```
Comment (bittersweetryan) [inline content/episodes.json]: I added some links in the doc, should I add them here?

File: content/episodes.json [hunk 2]
```diff
@@ -1,5 +1,122 @@
...
+        "from": "Ryan Anklam"
+      },
+      {
+        "title": "Vim Tabular",
+        "url": "https://github.com/godlygeek/tabular",
+        "from": "Ryan Anklam"
+      },
+      {
+        "title": "90s Hip Hop",
+        "url": "#",
```
Comment (bittersweetryan) [inline content/episodes.json]: spotify:user:bittersweetryan:playlist:2Fz6VlYg8ff2lZJro6SrPD

Comment (ryanburgess) [inline content/episodes.json]: Thanks! I was going to ask you about that :)

File: content/episodes.json (hunk 1 above)
Comment (ryanburgess) [inline content/episodes.json]: If you can add them here that'd be great! I just merged to master but feel free to add them.

**Pull request merged**